v22.2/create-external-connection.md

```diff
@@ -0,0 +1,170 @@
+---
+title: CREATE EXTERNAL CONNECTION
+summary: The CREATE EXTERNAL CONNECTION statement creates a new external connection for external storage. 
+toc: true
+docs_area: reference.sql
+---
+
+{% include feature-phases/preview.md %}
+
+{% include_cached new-in.html version="v22.2" %} You can use external connections to specify and interact with resources that are external from CockroachDB. With `CREATE EXTERNAL CONNECTION`, you define a name for an external connection while passing the provider URI and query parameters. [Backup](backup.html), [restore](restore.html), [import](import.html), [export](export.html), and [changefeed](create-changefeed.html) queries can interact with the defined external connection instead of a required, provider-specific URI. 
+
+`CREATE EXTERNAL CONNECTION` will validate the URI by writing, reading, and listing a test file to the external storage URI. If you're using a [KMS URI](take-and-restore-encrypted-backups.html), `CREATE EXTERNAL CONNECTION` will encrypt and decrypt a file. You'll find a `crdb_external_storage_location` file in your external storage as a result of this test. Each of the operations that access the external connection is aware of the raw URI that is parsed to configure, authenticate, and interact with the connection.
+
+The [privilege model](#required-privileges) for external connections means that you can delegate the creation and usage of external connections to the necessary users or roles.
+
+You can also use the following SQL statements to work with external connections:
+
+- [`SHOW CREATE EXTERNAL CONNECTION`](show-create-external-connection.html)
+- [`DROP EXTERNAL CONNECTION`](drop-external-connection.html)
+
+## Required privileges
+
+To create an external connection, a user must have the [`EXTERNALCONNECTION` system privilege](security-reference/authorization.html#supported-privileges). `root` and [`admin`](security-reference/authorization.html#admin-role) users have this system privilege by default and are capable of granting the `EXTERNALCONNECTION` privilege to other users and roles with or without the [`GRANT OPTION`](grant.html). 
```

Comment:
@adityamaru I know you will do this anyway, but can you take a close look over the privilege sections for these SQL statements... I'm not sure I have them 100%?